feat(sbb-radio-button-group): introduce radio-button-group component (#1431)
lyne-design-system/lyne-components: src/components/sbb-radio-button/readme.md

# sbb-radio-button

> Use the `sbb-radio-button` tag in combination with the `sbb-radio-button-group` component in order to display a radio input within the radio group.

Reviewer: ```suggestion
Use multiple `sbb-radio-button` components inside a `sbb-radio-button-group` component in order to display a radio input within a group.
```
